capstone/src/test/javascript/graphsTest.js

```diff
@@ -0,0 +1,17 @@
+describe('Data Table Construction', function() {
+  it('Will add correct columns/rows to DataTable from JSON', function() {
+    // TODO(ihsan314): Change this when format of JSON object is well-defined
+    const script = document.createElement('script');
+    script.src = 'https://www.gstatic.com/charts/loader.js';
+    script.defer = true;
```

Comment:
Are these lines necessary? Looks like the test passes without them...

Reply:
I think tests weren't passing when I didn't have those lines. I'll check again and confirm.

Reply:
Yea I confirmed tests are ok without them.